release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -4683,4 +4683,30 @@ $ oc adm release info 4.12.56 --pullspecs
 [id="ocp-4-12-56-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-12-57"]
+=== RHSA-2024:2782 - {product-title} 4.12.57 bug fix and security update
+
+Issued: 2024-05-16
+
+{product-title} release 4.12.57, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:2782[RHSA-2024:2782] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:2784[RHSA-2024:2784] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.12.57 --pullspecs
+----
+
+[id="ocp-4-12-57-bug-fixes"]
+==== Bug fixes
+
+* Previously, the `clear-irqbalance-banned-cpus.sh` script set an empty value for `IRQBALANCE_BANNED_CPUS` in `/etc/sysconfig/irqbalance`. As a result, the IRQs only balanced over the reserved CPUs. With this release, the `clear-irqbalance-banned-cpus.sh` script sets the banned mask to zeros on startup and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-31442[*OCPBUGS-31442*])
```

Comment:
Strictly speaking, `/etc/sysconfig/irqbalance` should be an adjective to a TBD noun.